solr/core/src/java/org/apache/solr/schema/ManagedIndexSchemaFactory.java

```diff
@@ -74,12 +75,12 @@ public void init(NamedList<?> args) {
     args.remove("mutable");
     managedSchemaResourceName = params.get(MANAGED_SCHEMA_RESOURCE_NAME, DEFAULT_MANAGED_SCHEMA_RESOURCE_NAME);
     args.remove(MANAGED_SCHEMA_RESOURCE_NAME);
-    if (SCHEMA_DOT_XML.equals(managedSchemaResourceName)) {
+    if (SCHEMA_DOT_XML.equals(managedSchemaResourceName)) { // TODO Still needed?
```

Comment:
Reading through this again...   I see that we have decided that schema.xml can't be the managed schema file...    Which I guess is fine, but if you wanted it to be?   Is tehre a use case for being able to override the default managed schema name?   I was wondering if it could just be hardcoded to "managed-schema.xml" (wiht the legacy fall back to "managed-schema".   Is there ever a reason to have a different name?

Reply:
I agree; I think a user should be able to configure it to be the same if they wish.  But I would make such a change in a separate issue, and also see that there is no problem with it.